features/step_definitions/proxy_steps.rb

```diff
@@ -64,30 +64,22 @@ def stub_deploy_calls!
 end
 
 Given(/^I add a new mapping rule with method "([^"]*)" pattern "([^"]*)" delta "([^"]*)" and metric "([^"]*)"$/) do |method, pattern, delta, metric|
-  click_on 'add-proxy-rule'
-  within(page.find('#sortable tr:last-child')) do
-    find("td.http_method select option[value='#{method}']").select_option
-    find('td.pattern input').set pattern
-    find('td.delta input').set delta
-    find('td.metric select').find(:xpath, "//*[.='#{metric}']").select_option
-  end
-end
-
-Given(/^I drag the last mapping rule to the position (\d+)$/) do |position|
-  within(page.find('#sortable')) do
-    last_index = all('tr').count
-    element = page.find("tr:nth-child(#{last_index}) .ui-sortable-handler")
-    target = page.find("tr:nth-child(#{position})")
-    element.drag_to(target)
+  click_on 'Add Mapping Rule'
+  within(page.find('form.proxy_rule')) do
+    find("select#proxy_rule_http_method option[value='#{method}']").select_option
+    find('input#proxy_rule_pattern').set pattern
+    find('input#proxy_rule_delta').set delta
+    find('select#proxy_rule_metric_id option', text: metric).select_option
   end
+  click_on 'Create Mapping Rule'
 end
 
 Given(/^I save the proxy config$/) do
   click_on 'proxy-button-save-and-deploy'
 end
 
 Then(/^the mapping rules should be in the following order:$/) do |table|
-  data = @provider.default_service.proxy.proxy_rules.includes(:metric).ordered
+  data = @provider.default_service.proxy.proxy_rules.includes(:metric).order_by(:delta)
```

Comment:
This is wrong 🙈 `the mapping rules should be in the following order` implies by `position`. Using this:
https://github.com/3scale/porta/blob/9ade227d1914f2bf580b65d259e15e176693c5b8/app/models/proxy_rule.rb#L7